src/components/map/hooks/use-update-bottom-sheet-when-selected-entity-changes.tsx

```diff
@@ -49,6 +51,9 @@ export const useUpdateBottomSheetWhenSelectedEntityChanges = (
     useBottomSheetContext();
   const analytics = useMapSelectionAnalytics();
   const navigation = useNavigation<RootNavigationProps>();
+  const hasReservationOrAvailableFareContract =
+    useHasReservationOrAvailableFareContract();
+  const {enable_vipps_login} = useRemoteConfigContext();
```

Comment:
Maybe should also add this on the dependency array for the hook, we are currently suppressing the warning using eslint, but since we use it in this hook `src/components/map/hooks/use-auto-select-map-item.tsx`, seems like it should also be added here.

Reply:
yes, done